<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>964</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>التهيئة</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="85">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="35,5,60">
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="styleSheet">
          <string notr="true">background-color:rgb(223, 221, 220);
color:rgb(0, 0, 0);

font: 18pt &quot;Mohammad Annoktah&quot;;</string>
         </property>
         <property name="text">
          <string>بيانات الحصص الخاصة بالترم</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,5,5,0,5,0,5,0">
         <item>
          <widget class="QPushButton" name="btnShowSession">
           <property name="styleSheet">
            <string notr="true">background-color:rgb(223, 221, 220);
color:rgb(0, 0, 0);
font: 14pt &quot;PT Bold Heading&quot;;</string>
           </property>
           <property name="text">
            <string>عرض</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnAddNewSession">
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>35</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:rgb(223, 221, 220);
color:rgb(0, 0, 0);
font: 14pt &quot;PT Bold Heading&quot;;</string>
           </property>
           <property name="text">
            <string>إضافة</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>icons/save.svg</normaloff>icons/save.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="txtNumberOfSessions">
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>35</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">	font: 10pt &quot;PT Bold Heading&quot;;</string>
           </property>
           <property name="text">
            <string>عدد الحصص في الاسبوع :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboSubjects">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>35</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;Mohammad Annoktah&quot;;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="styleSheet">
            <string notr="true">	font: 10pt &quot;PT Bold Heading&quot;;</string>
           </property>
           <property name="text">
            <string>الماده :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboClasses">
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>35</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;Mohammad Annoktah&quot;;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true">	font: 10pt &quot;PT Bold Heading&quot;;</string>
           </property>
           <property name="text">
            <string>الصف :</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="0">
          <widget class="QPushButton" name="btnMoveTermSessions">
           <property name="maximumSize">
            <size>
             <width>140</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:rgb(4, 0, 62);
color:rgb(255, 255, 255);
font: 15pt &quot;Mohammad Annoktah&quot;;</string>
           </property>
           <property name="text">
            <string>التالـــي</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>icons/arrowLeft.png</normaloff>icons/arrowLeft.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>35</width>
             <height>35</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="btnSkippingTermSessions">
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:rgb(4, 0, 62);
color:rgb(255, 255, 255);
font: 15pt &quot;Mohammad Annoktah&quot;;</string>
           </property>
           <property name="text">
            <string>تخطي</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>icons/arrow-left.png</normaloff>icons/arrow-left.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QTableWidget" name="tblTermSessions">
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">QTableWidget {
    background-color: #FFFFFF;
    alternate-background-color:rgb(0, 0, 0);
    font: 9pt &quot;PT Bold Heading&quot;;
    font-weight: bold;
 color:rgb(0, 0, 0)
}

QHeaderView::section {
    background-color:rgb(25, 21, 59);
    border: 1px solid rgb(0, 0, 0);
    font-weight: bold;
    padding: 4px;
    text-align: center;
	font: 10pt &quot;PT Bold Heading&quot;;
	color:rgb(255, 255, 255)
}

QTableWidget::item:selected {
    background-color:rgb(25, 21, 65);
    color:rgb(255, 255, 255);
}</string>
           </property>
           <column>
            <property name="text">
             <string>الرقم </string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>الصف</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>الماده</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>عدد الحصص في الاسبوع</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>العمليات</string>
            </property>
           </column>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QPushButton" name="btnCancel">
           <property name="styleSheet">
            <string notr="true">background-color:rgb(4, 0, 62);
color:rgb(255, 255, 255);
font: 15pt &quot;Mohammad Annoktah&quot;;</string>
           </property>
           <property name="text">
            <string>الغاء</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtNumberOfSessions</tabstop>
  <tabstop>btnAddNewSession</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>comboSubjects</tabstop>
  <tabstop>comboClasses</tabstop>
  <tabstop>btnMoveTermSessions</tabstop>
  <tabstop>tblTermSessions</tabstop>
  <tabstop>btnSkippingTermSessions</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
